src/library/basesqltablemodel.cpp

```diff
@@ -88,7 +88,7 @@ void BaseSqlTableModel::initSortColumnMapping() {
     m_columnIndexBySortColumnId[TrackModel::SortColumnId::SORTCOLUMN_DATETIMEADDED] = fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_DATETIMEADDED);
     m_columnIndexBySortColumnId
             [TrackModel::SortColumnId::SORTCOLUMN_LASTPLAYEDDATETIME] =
-                    fieldIndex(ColumnCache::COLUMN_LASTPLAYED_DATETIMEPLAYED);
+                    fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_DATETIMEPLAYED);
```

Comment:
Commits in the history that do not even build are bad for git bisect. Redo and force push.